test/Plutip.purs

```diff
@@ -136,10 +133,9 @@ suite = do
       withResource (startPlutipServer config)
         (stopChildProcessWithPort config.port) $ const do
         startRes <- startPlutipCluster config unit
-        startRes `shouldSatisfy` case _ of
-          ClusterStartupSuccess _ -> true
-          _ -> false
-        liftEffect $ Console.log $ "startPlutipCluster: " <> show startRes
+        liftEffect $ Console.log $ "startPlutipCluster: "
```

Comment:
Please remove the change to the logging on this line.

Reply:
I can't exactly because `PrivatePaymentKey` doesn't have a show instance, I can remove the `<ourKey> /\` part though

Reply:
Yes, please remove the `<ourkey>`